config/v1/types_image_content_policy.go

```diff
@@ -68,10 +69,11 @@ type RepositoryDigestMirrors struct {
 	// +kubebuilder:validation:Pattern=`^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])(:[0-9]+)?(\/[^\/:\n]+)*(\/[^\/:\n]+((:[^\/:\n]+)|(@[^\n]+)))?$`
 	Source string `json:"source"`
 	// allowMirrorByTags if true, the mirrors can be used to pull the images that are referenced by their tags. Default is false, the mirrors only work when pulling the images that are referenced by their digests.
+	// Setting different allowMirrorByTags values for the same source will be rejected.
 	// Pulling images by tag can potentially yield different images, depending on which endpoint
 	// we pull from. Forcing digest-pulls for mirrors avoids that issue.
 	// +optional
-	AllowMirrorByTags bool `json:"allowMirrorByTags,omitempty"`
+	AllowMirrorByTags *bool `json:"allowMirrorByTags,omitempty"`
```

Comment:
This is not backwards and forwards compatible, so this may not be added.  The default value is false, and must remain false.